external/webrtc-sdk.js

```diff
@@ -29473,20 +29482,14 @@ define('WebRTC_SDK/manager/RTCLocalMediaManager',['require','my.Class','undersco
                 deferred.resolve();
                 return deferred.promise;
             }
-            if(!_.isNull(stream) && !_.isUndefined(stream)) {
```

Comment:
No longer necessary to perform this check as we're returning early if the stream does not exist (see [here](https://github.com/mirainc/sdk-webrtc-meetings/pull/5/files#diff-7c9d62cf9e59419121e70349e9a94586R29468))